detectors/node/opentelemetry-resource-detector-alibaba-cloud/CHANGELOG.md

```diff
@@ -1,77 +1,5 @@
 # Changelog
 
-### Dependencies
```

Comment:
Are these removed manually? will we get more of them with the next releases?

Reply:
The removal of these `### Dependencies` sections were all done in this one commit if you want to review them together: https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2234/commits/4733ea4b037d0eb80ed0723414d9648a32423ba8

The issue is described here: https://github.com/open-telemetry/opentelemetry-js-contrib/issues/2207

tl;dr: This *was* a some bug in some old release-please-action that we were using. It isn't happening anymore, so I do **not** expect to get more of these sections.

Removing these sections here is effectively the "A quick fix ..." option described at https://github.com/open-telemetry/opentelemetry-js-contrib/issues/2207

Reply:
👍